packages/beacon-node/test/unit/network/doBeaconBlocksMaybeBlobsByRange.ts

```diff
@@ -0,0 +1,88 @@
+import sinon, {SinonStubbedInstance} from "sinon";
+import {expect} from "chai";
+import {peerIdFromString} from "@libp2p/peer-id";
+import {ssz, eip4844} from "@lodestar/types";
+import {createIBeaconConfig, createIChainForkConfig, defaultChainConfig} from "@lodestar/config";
+
+import {doBeaconBlocksMaybeBlobsByRange, ReqRespBeaconNode} from "../../../src/network/reqresp/index.js";
+import {BlockInputType} from "../../../src/chain/blocks/types.js";
+
+describe("doBeaconBlocksMaybeBlobsByRange", function () {
```

Comment:
Please always append `.test.ts` extension to test files

Reply:
ahh thats my bad, here is the fix: https://github.com/ChainSafe/lodestar/pull/4949